addons/lunch/static/src/js/lunch.js

```diff
@@ -0,0 +1,67 @@
+(function() {
+    "use strict";
+
+    var QWeb = openerp.web.qweb,
```

Comment:
Should probably be rebased and cleaned after the big merge in master about the web client.